tests/acceptance/stepDefinitions/debugContext.js

```diff
@@ -0,0 +1,5 @@
+const { When } = require('@cucumber/cucumber')
+
+When('pause for {int}', async function (duration) {
```

Comment:
Why commit this? Don't see any usage, did it help with debugging?:)

Reply:
its a helper to pause the test executions from the feature files... i think its pretty helpful

Reply:
Nice